controller.go

```diff
@@ -106,14 +119,16 @@ type driverData struct {
 type driverTable map[string]*driverData
 type networkTable map[types.UUID]*network
 type endpointTable map[types.UUID]*endpoint
-type sandboxTable map[string]*sandboxData
+type containerTable map[string]*sandboxData
```

Comment:
(Let's call Sandbox the new externally exposed entity and platformSandbox the existing internal entity which maps to a network namespace on Linux)

This is what I had in mind: 
Container <-> Sandbox  is a 1-on-1 relation (Sandbox will contain container specific configs, like etc/resolv.conf)
Container ->platformSandbox is not 1-on-1 relation, multiple container can point to the same platformSandbox (net=Container:<>, net=host for example)

The contention here is about controller to hold two different collections of "sandbox" related entities.
While it would be nice and clear to only have 3 collections in controller, each for each of the well-known externally exposed entities: networks, endpoints, sandboxes (the new ones)

As we agreed offline, I will try to see if I can remove the collection holding the platformSadboxes, and only have each Sandbox struct holding a pointer to the (shared or not) platformSandbox.
Who needs to iterate through the platformSandboxes will have to iterate through the Sandboxes and get the pointer to the platformSandbox.